Move default build directory within work directory (#429)
E3SM-Project/polaris: docs/developers_guide/updating_spack/testing/running_test_suites.md

5. **Set Up (and Auto-Build) the Suite**

    Omega will have already been built by the CTest utility in
    `build_omega/build_<machine>_<compiler>` under your Polaris branch (see
    {ref}`dev-build`).  To reuse that build when setting up the `omega_pr`
    suite, point `--component_path` at this directory, for example:

    ```bash
    polaris suite -c ocean -t omega_pr --model omega \
          -w /path/to/polaris_scratch/omega_pr_intel_openmpi \
          -p /path/to/polaris_branch/build_omega/build_<machine>_<compiler>
    ```

    If you omit `-p` and instead pass `--build`, Polaris will build Omega into
    the `build` subdirectory of the suite work directory
    (`${paths:base_work_dir}/build`).

    Notes:

>     - Don't use the`--branch` flag so that, the Omega submodule in your Polaris

Reviewer: ```suggestion
    - Using the`--branch` flag is not recommended so that the Omega submodule in your Polaris
```

Reviewer: I don't feel strongly about this at all.
